packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -40,8 +40,13 @@ export default class QuanticResult extends LightningElement {
 
   @track resultHasPreview = true;
 
+  /** @type {boolean} */
+  isHovered = false;
```

Comment:
What would you think of naming this attribute `actionsDisplayed`? It would reduce the coupling with the mouse input.

Reply:
The way I see it this property allows developer users to hook onto the "hovered" state when making custom result templates and this need might be for something else than displayed result actions. Also the result actions are tied to the insight panel use case right now but the quanticResult component is used in search as well. So keeping the property name more generic is more appropriate in my opinion.

Reply:
I agree with you that `isHovered` is not a bad name. The thing that bothers me a little is that it works only with the mouse input.

I just realized that the attribute is private. There is no problem using `isHovered` then. If we want to improve accessibility down the road, we can rename the attribute without breaking anything.